config/samples/events-handlers-sample-serving-only.yaml

```diff
@@ -104,7 +104,7 @@ spec:
       topic: "events-sample"
       authRequired: false
   sink:
-    uri: "http://openfunction.io.svc.cluster.local/default/sink-a"
+    uri: "http://sink-a.default.ofn.io/"
```

Comment:
change this to the cluster domain style? because user may not config the coredns

Reply:
has been modified locally, will update later